src/app/content/components/ContentExcerpt.tsx

```diff
@@ -37,7 +41,8 @@ const ContentExcerpt = styled((props: Props) => {
 
   const fixedContent = React.useMemo(() => flow(
     addTargetBlankToLinks,
-    (newContent) => fixRelativeURLs(newContent, excerptSource.url)
+    (newContent) => rebaseRelativeContentLinks(newContent, excerptSource.url),
+    (newContent) => rebaseRelativeResources(newContent, excerptSource.url)
```

Comment:
technically i think this one might not be necessary here, because the excerpts should be taken from the content after the paths are already fixed.

Reply:
so remove rebaseRelativeResources? I thought we needed both

Reply:
yea no its fine, i don't think it'll break anything